src/components/ChatEntry.js

```diff
@@ -1,22 +1,37 @@
-import React from 'react';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
+  const handleLike = () => {
+    props.toggleLike(props.id);
+  };
+
+  const senderClass = props.sender === 'Vladimir' ? 'local' : 'remote';
```

Comment:
Here are some things to consider:

- How could we dynamically get the names of the chat participants? In other words, how would we make this chat app apply to different participants like Mario and Luigi (or literally any other person 😄
-  What if the chat was starting and there weren't any messages yet?